fcrepo-exts/fcrepo-import-export
# improve shutdown impl to work on java 17+

# What this does

When I originally wrote the code that shutsdown the executor, I used reflection to get the info necessary to save the tasks that hadn't been processed. This will not work in Java 17+. This PR reworks the shutdown to no longer require reflection.

# How to test

Start an export and abort it while it's still running using ctrl+c. Then, verify the inflight tasks were logged.
